src/features/user/Widget/DonationLink/CustomSnackBar.tsx

```diff
@@ -0,0 +1,47 @@
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+import i18next from '../../../../../i18n';
+import { ReactElement } from 'react';
+import styled from '@emotion/styled';
```

Comment:
`styled` should be imported from mui, not emotion, as that gives access to the mui theme (otherwise `theme.palette` is not available)

`import styled from '@mui/material/styles/styled';`